### PR TITLE
assigned task done

### PR DESCRIPTION
**Pull request by vishnusandeep969**
change See our work → We can build anything you want

change this “The new Style for 2021 is here. Like it? BUY THEME NOW!” →  'See our work ' (make it like the buy theme now  style)

About us, the menu button direction is now to the portfolio section. (check that) change direction to the about us section.

change the website title → Brainchild Services

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,10 +184,10 @@ <h1> We make our <br>people and products<br>sensational.</h1>
             <div class="row m-0">
                 <div class="col-12 p-0">
                     <div class="pa-stripe-heading">
-                        <h1 class="pa-h1-v1 pa-bright fadeInLeftDelay03Duration10"><strong>See our work</strong></h1>
+                        <h1 class="pa-h1-v1 pa-bright fadeInLeftDelay03Duration10"><strong>We can build anything you want</strong></h1>
                     </div>
                     <div class="pa-stripe-subheading">
-                        <p class="pa-p-v2 pa-bright fadeInLeftDelay03Duration10">The new Style for 2021 is here. Like it? <a href="https://themeforest.net/item/particle-modern-tech-startup-html-template/20078383?ref=Skilltech" target="_blank">BUY THEME NOW!</a></p>
+                        <p class="pa-p-v2 pa-bright fadeInLeftDelay03Duration10"><a href="https://themeforest.net/item/particle-modern-tech-startup-html-template/20078383?ref=Skilltech" target="_blank">SEE OUR WORK!</a></p>
```
Comment (wandercode) [inline index.html]: please change the href address null -->#

**Review by wandercode — CHANGES_REQUESTED** [review]

**Comment by vishnusandeep969** [comment]
changes done

**Comment by wandercode** [comment]
LGTM 👍

**Review by VishnuKulakkattu — APPROVED** [review]
LGTM

**Review by wandercode — APPROVED** [review]

**Pull request merged**